Nodejs/Product/Npm/SPI/NpmController.cs

```diff
@@ -100,18 +95,19 @@ internal class NpmController : AbstractNpmLogSource, INpmController {
         }
 
         public void Refresh() {
-            RefreshAsync().ContinueWith(t => {
-                var ex = t.Exception;
+            try {
+                RefreshImplementation();
+            } catch (Exception ex) {
                 if (ex != null) {
                     OnOutputLogged(ex.ToString());
 #if DEBUG
                     Debug.Fail(ex.ToString());
 #endif
                 }
-            });
+            }
         }
 
-        public async Task RefreshAsync() {
+        private void RefreshImplementation() {
```

Comment:
The main reason for this change was not refactoring, but because when I removed `NpmBinCommand`, `RefreshAsync` can no longer be an async method. It's a compiler error to mark it `async` since it does not have any async behavior.

I chose to rename this method and remove the async behavior, but an alternative could possibly be making `RefreshImplementation` return an async void or similar. What do you think?